HISTORY.md

```diff
@@ -1,5 +1,13 @@
 # History
 
+## 2.0.x
+
+- Integrate poetry dependency management substituting conda development environments (following MCH python templates)
+- Automate distributable package generation using poetry
+- Configure CI/CD pipeline to run in MCH Jenkins server
+- Containerize pyflexplot tool to be able to run it within a container
+- Integrate AWS deployment
+
 ## 1.0.10 (2023-XX-XX)
 
 - First release on PyPI.
```

Comment:
Has v1.x ever been released on PyPI? Not as far as I now, unless @nestabur made a release of 1.0.10 on PyPI (as the date is not set, probably not?).

Reply:
There is no package in PyPI repository, that entry in History is an old one ... shoulw we remove it?

Reply:
Edited HISTORY.md https://github.com/MeteoSwiss-APN/pyflexplot/pull/41/commits/f4dd6891e2f220ef55f101c03c0a924c02319148